vm/src/builtins.rs

```diff
@@ -716,11 +716,13 @@ fn builtin_setattr(
 // builtin_slice
 
 fn builtin_sorted(vm: &VirtualMachine, mut args: PyFuncArgs) -> PyResult {
-    arg_check!(vm, args, required = [(iterable, None)]);
-    let items = vm.extract_elements(iterable)?;
+    let iterable = args.take_positional();
+    if iterable.is_none() {
+        return Err(vm.new_type_error("sorted expected 1 arguments, got 0".to_string()));
+    }
+    let items = vm.extract_elements(&iterable.unwrap())?;
```

Comment:
Though it is already merge, we conventionally prefer to use this form to avoid unwrap when possible:
```suggestion
    let iterable = args.take_positional().unwrap_or_else(|| Err(vm.new_type_error("sorted expected 1 arguments, got 0".to_string())))?;
    let items = vm.extract_elements(&iterable)?;
```